fix for missing data in parsing hrf

brokenelevator opened the pull request:
since properties are saved in lower case they should be searched for in lower case.
now levels for form coach and financial director in the misc tab(not to mention database) are correct.
staff list was ok but levels below were invalid.

akasolace commented:
was not amare of that bug. I'll have a look and merge it asap.
Thanks for the new contrib !
for my info, what is your id on HT ?

brokenelevator commented:
well i was aware if it for quite a while but the solution was not trivial and I didn't mind the bug.

thanks for the merges.
 
I'm using HO since the beginning(that is 2010), it's a great piece of software. for some time I was using custom version of it based on the latest revision from sourceforge. added few things, fixed some bugs, like support player, some new match event crash and alike. those are now fixed in main HO, I'm using my own workarounds for now. I tried to approach Bingeling once or twice but that didn't go well. I tried to help on the forums once in a while but with the bugs I couldn't do much.
Now that HO is on github, even though I would rather it stay on gitlab, contributing is rather easy so when I saw the bug again I figured I will fix it and send pull request. You already merged one(by now three).
The one with duplicated country I saw when I was preparing to put my custom changes to the new version and I don't like broken things. It(the duplicate) probably didn't mess anything up but I don't like errors. When I'm done and those work with the new version I may upstream them if they don't break HT CHPP rules or something else.
Later I plan to update translation in my language and maybe(if not done already) add the new substitutions to HO (as per issue in github). So If you could add it to poeditor it would be nice, unless you want me to PR it.

Sorry for the long message, not everyone likes rather long messages.

My HT name is the same as here. I run AC Goa, Poland, division IV.

akasolace commented:
@brokenelevator 
Thank you for providing the feedback !  You contribution are more than welcome. 

> I'm using HO since the beginning(that is 2010)

Myself I have contributed to HO around 2011 and then I dropped HO and HT for a few years


> I tried to approach Bingeling once or twice but that didn't go well. I tried to help on the forums once in a while but with the bugs I couldn't do much.

Bingeling did not want to be involved anymore, this is why when I wanted to push a bugfix I ended up forking the project instead :-)

> Now that HO is on github, even though I would rather it stay on gitlab

The problem I had with GitLab was that it does not allows for large artefacts, so I would have need to use SF on top of Gitlab just to distribute the artefacts. I did not think this was the right approach. Also I have been only using GitLab until HO but I should say that GitHub is extremly well polished and responsive. 

> When  I'm done and those work with the new version I may upstream them if they don't break HT CHPP rules or something else.

As said your contributions are welcome. 

>  add the new substitutions to HO (as per issue in github)

This one is the big one and I would like 1.436 to be focus on improving the Lineup experince and the prediction ratings - this is the core of HO in the end.  I have some ideas and it looks like we are now 3 motivated developers so the workload will stay reasonable . We will have to coordinate about it and split the work to be efficient !

> Later I plan to update translation in my language and maybe(if not done already) add the new substitutions to HO (as per issue in github). So If you could add it to poeditor it would be nice, unless you want me to PR it.

I would add it to PoEditor tonight

On another topic, Mistermax is now seting up a discord chat room for the developer I will ask hi to invite you as well.  It will be a low frequency chat but will help to communicate without using bug ticket or PR :-)

> My HT name is the same as here. I run AC Goa, Poland, division IV.

Welcome onboard brokenelevator !

brokenelevator commented:
@akasolace as I already mentioned I'm happy to help :)
I don't plan to drop HT, I gave my team a lot of time and effort. Building a team is not an easy business as you probably know. But if/when I leave I most likely won't come back. This is my team and I don't want another.

> Bingeling did not want to be involved anymore, this is why when I wanted to push a bugfix I ended up forking the project instead :-)

I figured and also thought that fork would be needed but didn't want to be the boss. So for now this situation suits me :)

> The problem I had with GitLab was that it does not allows for large artefacts

I think it does. I mean I haven't used it but it has a feature called `Git Large File Storage`.

About CHPP
The problem may be that I don't know the rules or source of some of the code I used.  Will deal with it when the need comes.

> I have some ideas and it looks like we are now 3 motivated developers so the workload will stay reasonable . We will have to coordinate about it and split the work to be efficient !

Indeed it looks like it. We will figure it out.  With the predictions we need some data because coding is one thing, estimation, prediction and stuff is whole another case.

> I would add it to PoEditor tonight

Thanks, I saw it is there already. I will get to it some time soon. It is not like I have to do all of it (the 29%) at once.

> Mistermax is now seting up a discord chat room for the developer I will ask hi to invite you as well.

That is great. Link me or something when I'm added.

> Welcome onboard brokenelevator !

Thank you very much. It is great to be here. Will see how it goes.

akasolace commented:
> I think it does. I mean I haven't used it but it has a feature called Git Large File Storage.
GLFS is solving another problem, it is used for versioning of large file. If the project requires a large binary file and that file is modified often, instead of having local copy of that file, and big usage of your git repository, GLFS ensure pointers to external file to keep your git repository clean

In my case, the issue was to offer binaries to end user (downloadable executable) and GitLab size restriction was too tight for this.

> That is great. Link me or something when I'm added.

the discord link is:   https://discord.gg/CAE4Un6
